jl4-lsp/src/LSP/L4/Viz/VizExpr.hs

```diff
@@ -19,7 +19,7 @@ newtype RenderAsLadderInfo = MkRenderAsLadderInfo
 type Unique = Int
 -- | TODO: Consider renaming this to something other than `Name`
 data Name = MkName
-  { unique :: Unique   -- ^ for checking whether, e.g., two BoolVar IRNodes actually refer to the same proposition.
+  { unique :: Unique   -- ^ for checking whether, e.g., two UBoolVar IRNodes actually refer to the same proposition.
```

Comment:
The changes in this file amount just to renaming BoolVar to UBoolVar etc to align with the more explicit distinction between BoolValues and UBoolValues on the frontend